packages/compass-databases-navigation/src/item-action-controls.tsx

```diff
@@ -123,6 +124,7 @@ export const ActionControls: React.FunctionComponent<{
           <IconButtonSmall
             key={action}
             glyph={icon}
+            mode="hovered"
```

Comment:
Is there a case there this `IconButtonSmall` is used that isn't hovered?

Reply:
Aah good point. I removed it. Only used on the icons themselves.